Microsoft.Toolkit.Uwp.Services/Services/Twitter/Tweet.cs

```diff
@@ -21,12 +21,6 @@ public class Tweet : Toolkit.Parsers.SchemaBase, ITwitterResult
         [JsonProperty("created_at")]
         public string CreatedAt { get; set; }
 
-        /// <summary>
-        /// Gets or sets the geographic data (latitude and longitude)
-        /// </summary>
-        [JsonProperty("geo")]
-        public TwitterGeoData GeoData { get; set; }
```

Comment:
This is a breaking change. We cannot just remove properties. They need to be marked as deprecated and removed at a future major version